rnd/docker-compose.yml

```diff
@@ -134,13 +143,18 @@ services:
     build:
       context: ../
       dockerfile: rnd/market/Dockerfile
+    develop:
+      watch:
+        - path: ./
+          target: rnd/market/
+          action: rebuild
     depends_on:
       - postgres
       - migrate
     environment:
       - DATABASE_URL=postgresql://agpt_user:pass123@postgres:5432/agpt_local
     ports:
-      - "8015:8015"
+      - "8015:8000"
```

Comment:
Why is this needed ?

Reply:
Instead of using an env var to get unicorn to use a different port, I'm using docker port mapping instead. Its needed to expose it to localhost